### PR TITLE
(1 of 2) rerun path subtyping due to LGAT subtyping changes

### PR DESCRIPTION
**Pull request by jharenza**
#### What scientific question is your analysis addressing?
Per this [PR comment](https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/924#discussion_r569781943), I am rerunning the pathology subtyping module to reflect all of the changes that occurred with #925, prior to updating the H3 K28 subtype for 7316-2557 in #924

#### What was your approach?
rerun module

#### What GitHub issue does your pull request address?
[PR comment](https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/924#discussion_r569781943)

#### Which areas should receive a particularly close look?
NA

#### Is there anything that you want to discuss further?
no

#### Is the analysis in a mature enough form that the resulting figure(s) and/or table(s) are ready for review?
yes

#### What types of results are included (e.g., table, figure)?
table

#### What is your summary of the results?
update looks as expected

#### Reproducibility Checklist

- [ ] The dependencies required to run the code in this pull request have been added to the project Dockerfile.
- [ ] This analysis has been added to continuous integration.

#### Documentation Checklist

- [x] This analysis module has a `README` and it is up to date.
- [x] This analysis is recorded in the table in `analyses/README.md` and the entry is up to date.
- [x] The analytical code is documented and contains comments.

### CODE REVIEW

File: analyses/molecular-subtyping-pathology/01-compile-subtyping-results.Rmd [hunk 1]
```diff
@@ -227,12 +227,14 @@ lgat_results_df <- bind_exp_strategies(lgat_results_df) %>%
...
-                                          molecular_subtype == "LGG, BRAF wildtype"~ "Low-grade glioma/astrocytoma, BRAF wildtype", 
-                                          TRUE ~NA_character_),
-                  broad_histology = "Low-grade astrocytic tumor",
-                  short_histology = if_else(pathology_diagnosis == "Ganglioglioma", "Ganglioglioma", "LGAT"),
+  mutate(integrated_diagnosis = case_when(grepl("LGG", molecular_subtype) ~ paste0("Low-grade glioma/astrocytoma, ", molecular_subtype),
+                                          TRUE~ NA_character_),
+         #remove "LGG, To be classified" from integrated dx
+         integrated_diagnosis = ifelse(molecular_subtype == "LGG, To be classified", NA_character_, integrated_diagnosis),
+         #delete LGG string from integrated dx
+         integrated_diagnosis = case_when(grepl("LGG", integrated_diagnosis) ~ str_replace(integrated_diagnosis, "LGG, ", "")),
```
Comment (jharenza) [inline analyses/molecular-subtyping-pathology/01-compile-subtyping-results.Rmd:235]: There is probably a better, more clever way to do this when pasting above in line 230, but just wasn't sure how! Open to suggestions.

File: analyses/molecular-subtyping-pathology/01-compile-subtyping-results.Rmd [hunk 2]
```diff
@@ -227,12 +227,14 @@ lgat_results_df <- bind_exp_strategies(lgat_results_df) %>%
...
+         broad_histology = "Low-grade astrocytic tumor",
+         short_histology = if_else(pathology_diagnosis == "Ganglioglioma", "Ganglioglioma", "LGAT"),
          Notes = if_else(!is.na(integrated_diagnosis), "Updated via OpenPBTA subtyping", Notes))
```
Comment (jaclyn-taroni) [inline analyses/molecular-subtyping-pathology/01-compile-subtyping-results.Rmd:238]: You can use `case_when()` to set `integrated_diagnosis` to `NA` when `molecular_subtype` is `LGG, To be classified` and then use `str_replate()` for all other cases:

```suggestion
mutate(integrated_diagnosis = case_when(
            str_detect(molecular_subtype, "To be classified") ~ NA_character_,
            TRUE ~ str_replace(molecular_subtype, "LGG, ", "Low-grade glioma/astrocytoma, ")
          ),
         broad_histology = "Low-grade astrocytic tumor",
         short_histology = if_else(pathology_diagnosis == "Ganglioglioma", "Ganglioglioma", "LGAT"),
         Notes = if_else(!is.na(integrated_diagnosis), "Updated via OpenPBTA subtyping", Notes)
  )
```

**Review by jaclyn-taroni — APPROVED** [review]
👍  I would test my suggested change (I did, but we should make sure you do, too) and if it looks good, let's save a few lines -> get this merged 🏎️

File: analyses/molecular-subtyping-pathology/01-compile-subtyping-results.Rmd (hunk 2 above)
Comment (jharenza) [reply analyses/molecular-subtyping-pathology/01-compile-subtyping-results.Rmd]: thanks, updated and works - you're a whiz :) !

**Pull request merged**